### DISCUSSION TITLE
Nexus Lifecycle found Security vulnerabilities in Allure Report

### DISCUSSION THREAD

**Discussion by abulat** [allure-framework/allure2#2620] [Bug Reports] [answered]
Our company is trying to integrate Allure Report extension to Azur Devops. We scanned it with Nexus Lifecycle and identified 3 Level 9 vulnerabilities in allure-commandline: 2.20.1, mockery : 1.7.0 and mockery: 2.1.0: 
![thumbnail_image001](https://github.com/allure-framework/allure2/assets/3479748/671b41a7-f7bc-4f05-ab70-6ac4a68b7d8f)

Scanning report:
[qameta.allure-azure-pipelines-1.4.1 (LVL 9).pdf](https://github.com/user-attachments/files/16050496/qameta.allure-azure-pipelines-1.4.1.LVL.9.pdf)

would it be possible to fix security vulnerabilities and release a new version?

**Comment by baev** [comment]
@abulat, we're looking into the issue. Stay tuned for the updates

**Comment by epszaw** [comment, marked as answer]
Most vulnerabilities were solved. The latest version (`1.4.2`) includes the required changes. Could you please check the fix?

**Reply by abulat** [reply]
Thank you for a quick response. 
I requested a new security scan. Will let you know as soon as it is finished.

**Comment by abulat** [comment]
I am confirming that mentioned vulnerabilities disappear in a new version of extension (1.4.2). 

Thank you very much for a very quick reaction.
